var/spack/repos/builtin/packages/xeniumranger/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
If you do update the package, can you change the ending copyright year to 2024?  Spack checks will not complain about using 2023 right now because they allow the previous year.

Reply:
Done in commit below